ats/pkb/pkb.py

```diff
@@ -0,0 +1,33 @@
+def follows(list, s1, s2):
+    all_children = set(list.children)
+    pool = all_children.copy()
+    while pool:
+        child = pool.pop()
+        all_children.add(child)
+        pool.update(child.children)
+        for i, ch in enumerate(child.children):
+            if s1 in child.children.__str__() and s2 in child.children.__str__():
+                return True
+            else:
+                return False
```

Comment:
To wszystko można sprawdzić po prostu porównując indeksy `stmtLst.nodes.statements.index(s1) + 1 == stmtLst.nodes.statements.index(s2)`